shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/BridgeExecutorServiceImpl.java

```diff
@@ -173,6 +175,15 @@ public void createOrUpdateBridgeExecutorSecret(BridgeExecutor bridgeExecutor, Pr
         expected.getData().put(GlobalConfigurationsConstants.KAFKA_ERROR_TOPIC_ENV_VAR, Base64.getEncoder().encodeToString(processorDTO.getKafkaConnection().getErrorTopic().getBytes()));
         expected.getData().put(GlobalConfigurationsConstants.KAFKA_GROUP_ID_ENV_VAR, Base64.getEncoder().encodeToString(bridgeExecutor.getSpec().getId().getBytes()));
 
+        Optional<KafkaConnectionDTO> optOutboundKafkaConnection = processorDTO.parseKafkaOutgoingConnection();
+        optOutboundKafkaConnection.ifPresent(outboundKafkaConnection -> {
+            LOGGER.info("Outbound Kafka Connection configured: {}", outboundKafkaConnection);
```

Comment:
We should not log this DTO. It contains connection information.
Please remove it or log only part of it (or tune `KafkaConnectionDTO.toString()` to hide sensible information).

Reply:
`KafkaConnectionDTO.toString()` already hides the secret part